design/client.md

```diff
@@ -0,0 +1,12 @@
+# Client
+
+Since containerd uses gRPC, it should be easy to implement 3rd-party containerd client in almost any language.
```

Comment:
Want to add a TODO here with more information about how containerd is a client-heavy design.

Reply:
Are we assuming the reader should know the pros and cons of gRPC and how it lends itself to implementing 3rd-party containerd clients? It may be helpful to provide an example.

Reply:
We've already made the choice. ;)

@estesp has a talk somewhere where he discusses the decision. There are also some old discussions about how avoiding http specification time speeds up implementation.

A link would be good to `grpc.io.